### ISSUE TITLE
파이썬 초보인데 제가 입력한 코드 어드바이스 좀 해주세요.

### ISSUE THREAD

**Issue by Jeongyoungwang** [devkingsejong/python-PowerNad#2]
import sys
import io
sys.stdout = io.TextIOWrapper(sys.stdout.detach(), encoding = 'utf-8')
sys.stderr = io.TextIOWrapper(sys.stderr.detach(), encoding = 'utf-8')

from powernad.Connector.restapi import RestApi
from powernad.Object.RelKwdStat.RelKwdStatObject import RelKwdStatObject
from typing import List
RelKwdStatObjectList = List[RelKwdStatObject]

class RelKwdStat: #클래스 선언

#클래스 초기화
    def __init__(self, base_url: str, api_key: str, secret_key: str, customer_id: int): #객체 초기화
        self.r = RestApi('https://api.naver.com', '0100000' #Constructor 생성자
, 'ABCDE==', 123456)  #튜플??

    def get_rel_kwd_stat_list(self, siteId: str, biztpId: int = None, hintKeywords: str = None, event: int = None,
                              month: int = None, showDetail: int = None) -> RelKwdStatObjectList:
#딕셔너리
        query = {'siteId': siteId, 'biztpId': biztpId, 'hintKeywords': hintKeywords,
                                                             'event': event, 'month': month, 'showDetail': showDetail}
        result = self.r.get('/keywordstool', query)
        result = result['keywordList'] #리스트
        relstat_list = [] #빈 리스트

        for arr in result: #반복문
            relstat = RelKwdStatObject(arr)
            relstat_list.append(relstat)

        return relstat_list

import json

class RelKwdStatObject:
    def __init__(self, json_def):

        if type(json_def) is str:
            json_def = json.loads(json_def)
        s = json_def

        self.relKeyword = None if 'relKeyword' not in s else s['relKeyword']
        self.monthlyPcQcCnt = None if 'monthlyPcQcCnt' not in s else s['monthlyPcQcCnt']
        self.monthlyMobileQcCnt = None if 'monthlyMobileQcCnt' not in s else s['monthlyMobileQcCnt']
        self.monthlyAvePcClkCnt = None if 'monthlyAvePcClkCnt' not in s else s['monthlyAvePcClkCnt']
        self.monthlyAveMobileClkCnt = None if 'monthlyAveMobileClkCnt' not in s else s['monthlyAveMobileClkCnt']
        self.monthlyAvePcCtr = None if 'monthlyAvePcCtr' not in s else s['monthlyAvePcCtr']
        self.monthlyAveMobileCtr = None if 'monthlyAveMobileCtr' not in s else s['monthlyAveMobileCtr']
        self.plAvgDepth = None if 'plAvgDepth' not in s else s['plAvgDepth']
        self.compIdx = None if 'compIdx' not in s else s['compIdx']

개발왕세종님께서 알려준 코드 있는대로 붙이고, API 관리센터에서 얻은 정보들 얻고, 
코드 입력했는데 응답이 없네요 제가 뭔가 추가로 코드 입력 할게 있을까요??

**Comment by devkingsejong** [comment]
과제용으로 만든건데 관심 가져 주셔서 감사합니다.

해당부분 아래와 같이 사용해 주시면 됩니다.

```
rel = RelKwdStat(client.BASE_URL, client.API_KEY, client.SECRET_KEY, client.CUSTOMER_ID)
t = rel.get_relkwd_stat_list('bsn-lorem-ipsum)
print(t[0].monthlyAveMobileClkCnt)
```
